weather-card/weather-card.js

```diff
@@ -248,21 +244,19 @@ class WeatherCard extends LitElement {
           }
         
           .variations {
-            display: inline-block;
+            display: flex;
+            flex-flow:row wrap;
             font-weight: 300;
             color: var(--primary-text-color);
             list-style: none;
             margin-left: -2em;
             margin-top: 4.5em;
           }
-        
-          .variations.right {
-            position: absolute;
-            right: 1em;
-            margin-left: 0;
-            margin-right: 1em;
+
+          .variations li{
+            flex-basis: 50%; 
```

Comment:
I don't like that the right column is now more in the middle than on the right. I would like the space from the edge the same for both left and right.

Reply:
Yes, you are right, There is a problem with moving it to be the same, I was trying several methods but no success for me :( I will try to think about new options.

Reply:
Unfortunately, I could not bring it back the alignment the same position as before. 
So this Bug fix is "Give and Take".
Because this card not usable in RTL language as is now.
This is the problem in RTL:
![image](https://user-images.githubusercontent.com/10164935/51070568-0cd32300-164c-11e9-9e6a-e1a67040ffc1.png)

So basically we have 2 options: 
1. Approve this request and the card will look in LTR like this:
![image](https://user-images.githubusercontent.com/10164935/51070583-31c79600-164c-11e9-9921-b72b09a09a45.png)
and in RTL like this:
![image](https://user-images.githubusercontent.com/10164935/51070587-3db35800-164c-11e9-8d45-59a2fef0c6b6.png)

2.OR put all the variations in one line and it will look like this:
LTR:
![image](https://user-images.githubusercontent.com/10164935/51070728-da76f500-164e-11e9-89d6-66e91c21f37c.png)
RTL:
![image](https://user-images.githubusercontent.com/10164935/51070762-36417e00-164f-11e9-9df9-a2fd272247c6.png)

*Still the Days need to be "flipped" for full RTL support, but this one is for next time

Tell me what you think 👍